src/main/java/picard/arrays/CombineGenotypingArrayVcfs.java

```diff
@@ -0,0 +1,270 @@
+package picard.arrays;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.util.CloseableIterator;
+import htsjdk.samtools.util.IOUtil;
+import htsjdk.samtools.util.Log;
+import htsjdk.samtools.util.ProgressLogger;
+import htsjdk.variant.variantcontext.CommonInfo;
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.GenotypesContext;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.variantcontext.VariantContextUtils;
+import htsjdk.variant.variantcontext.writer.Options;
+import htsjdk.variant.variantcontext.writer.VariantContextWriter;
+import htsjdk.variant.variantcontext.writer.VariantContextWriterBuilder;
+import htsjdk.variant.vcf.VCFConstants;
+import htsjdk.variant.vcf.VCFFileReader;
+import htsjdk.variant.vcf.VCFHeader;
+import htsjdk.variant.vcf.VCFHeaderLine;
+import htsjdk.variant.vcf.VCFUtils;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.help.DocumentedFeature;
+import picard.PicardException;
+import picard.arrays.illumina.InfiniumVcfFields;
+import picard.cmdline.CommandLineProgram;
+import picard.cmdline.StandardOptionDefinitions;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.TreeMap;
+import java.util.TreeSet;
+
+/**
+ * A simple program to combine multiple genotyping array VCF files into one VCF
+ *
+ */
+@CommandLineProgramProperties(
+        summary = CombineGenotypingArrayVcfs.USAGE_DETAILS,
+        oneLineSummary = "Program to combine multiple genotyping array VCF files into one VCF.",
+        programGroup = picard.cmdline.programgroups.GenotypingArraysProgramGroup.class
+)
+@DocumentedFeature
+public class CombineGenotypingArrayVcfs extends CommandLineProgram {
+    static final String USAGE_DETAILS =
+            "CombineGenotypingArrayVcfs takes one or more VCF files, as generated by GtcToVcf " +
+                    "and combines them into a single VCF. " +
+                    "The input VCFs must have the same sequence dictionary and same list of variant loci. " +
+                    "The input VCFs must not share sample Ids. " +
+                    "<h4>Usage example:</h4>" +
+                    "<pre>" +
+                    "java -jar picard.jar VcfToAdpc \\<br />" +
```

Comment:
I think this should be 
```suggestion
                    "java -jar picard.jar CombineGenotypingArrayVcfs \\<br />" +
```

Reply:
Yeah, I cut and pasted an example usage obviously.